R/read_nmext.R

```diff
@@ -1,17 +1,28 @@
-#' Title
+# This code is strongly based on the read_nmext function of mrgsolve (nmxml.R) (Thanks to Kyle T Baron)
+# The code was slightly adjusted to check for multiple tables and also extract SE (ITERATION == 1000000001)
+# The output was also slightly adjust to fit ggPMX output (df and df2)
+# Addition of param, "quiet", (option of msg function, from xpose package)
+
+#' Reads .ext files generated by NONMEM
```

Comment:
I would add a @author section here and give him credit, and you credit as well.  Also note where the differences are in this function.  One last point is that I would name it something different to avoid a name clash with mrgsolve (in case they use both).

Reply:
Added @author (also in pmx_nm), renamed function to "read_extfiles", added code line numbers to indicate where the changes are